csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DependencyManager.Nuget.cs

```diff
@@ -479,27 +579,63 @@ private IEnumerable<string> GetFeeds(string nugetConfig)
                     continue;
                 }
 
-                yield return url;
+                if (!string.IsNullOrWhiteSpace(url))
+                {
+                    yield return url;
+                }
             }
         }
 
-        private HashSet<string> GetAllFeeds(List<FileInfo> allFiles)
+        private (HashSet<string>, HashSet<string>) GetAllFeeds(List<FileInfo> allFiles)
         {
+            IList<string> GetNugetFeeds(string nugetConfig)
+            {
+                logger.LogInfo($"Getting Nuget feeds from '{nugetConfig}'...");
```

Comment:
Consider adding the logging directly in `Dotnet.GetNugetFeeds` to avoid the wrapping here.